internal/config/config.go

```diff
@@ -31,6 +31,7 @@ const (
 	hcloudLoadBalancersNetworkZone           = "HCLOUD_LOAD_BALANCERS_NETWORK_ZONE"
 	hcloudLoadBalancersDisablePrivateIngress = "HCLOUD_LOAD_BALANCERS_DISABLE_PRIVATE_INGRESS"
 	hcloudLoadBalancersUsePrivateIP          = "HCLOUD_LOAD_BALANCERS_USE_PRIVATE_IP"
+	hcloudLoadBalancersUseNodeIP             = "HCLOUD_LOAD_BALANCERS_USE_NODE_IP"
```

Comment:
I am not sure about the naming or the option, this seems to target robot server only right?